docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,126 @@
+# Release 365 (XX Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for time travel queries. ({issue}`8773`)
+* Add [`contains`](ip_address_contains) function to check whether a CIDR contains an IP address. ({issue}`9654`)
+* Report connector metrics in `EXPLAIN ANALYZE VERBOSE`. ({issue}`9858`)
+* Report operator input distribution in ``EXPLAIN ANALYZE VERBOSE``. ({issue}`10133`)
+* Allow executing ``INSERT`` or ``DELETE`` statements on tables restricted with a row filter. ({issue}`8856`)
+* Improve performance of inequality joins. ({issue}`9307`)
+* Improve performance of joins with small build side. ({issue}`9851`)
```

Comment:
with a small?

Reply:
Actually, I'm going to rephrase it as: "Improve performance of joins involving a small table on one side". The notion of "build side" is not something that users generally understand.